scripts/cli_parser.py

```diff
@@ -0,0 +1,352 @@
+import argparse
+
```

Comment:
add high-level comment about what this script is for

Reply:
9fe8324